images/hook/entrypoint.sh

```diff
@@ -13,7 +13,11 @@ if [ $1 = "update" ]; then
     sed -i 's/localhost/cassandra.default.svc.cluster.local/' pithoscfg.yaml
     kubectl apply -f pithoscfg.yaml
 
-    rig delete configmap/cassandra-cfg --force
+    if kubectl get configmap cassandra-cfg > /dev/null 2>&1
+    then
+        kubectl delete configmap cassandra-cfg
```

Comment:
Why are we switching from `rig` to `kubectl` ?

Reply:
it fails with `ERROR: unsupported resource: configmap`, I will look at later why it fails. For now I want this thing works here...